### ISSUE TITLE
[FEATURE] Events map

### ISSUE THREAD

**Issue by Giuliano1993** [EddieHubCommunity/BioDrop#8932]
### Description

I was looking at the map tab, and seeing other BioDrop users in the surroundings is great. 

What about if we added a map to the events tab too? 

It could be useful to catch with eye where are the venues listed by the users, so if one is interested in the nearest ones, one can simply zoom the map in ( same mechanic as the users map ).

To help the search for the events, a time span could be added as an optional input. 

What do you think? 

### Do you want to work on this issue?

None

**Comment by SaraJaoude** [comment]
Thanks for raising! Currently there is a PR which is similar - whereby we are adding the events to the existing map https://github.com/EddieHubCommunity/BioDrop/pull/8093

It would be best to keep to one map only on the platform. 

I have added the `future consideration` label to this issue and we can revisit this once https://github.com/EddieHubCommunity/BioDrop/pull/8093 has been merged and update the description accordingly.
